rma/models/stock_warehouse.py

```diff
@@ -73,7 +73,7 @@ def _get_picking_type_create_values(self, max_sequence):
                 'name': _('RMA Receipts'),
                 'code': 'incoming',
                 'use_create_lots': True,
```

Comment:
I think this should be false, as you don't create lots for rma

Reply:
I'd say it's mostly disabled by the `use_existing_lots` setting being `True`, but maybe it's better to leave it to `False` indeed.

Reply:
They are not excluding in v12 AFAIK.

Reply:
It expects `use_existing_lots` to be `False` here, for example: https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_move.py#L490

Anyway, it's not very intuitive I must say. So I agree to set it to `False` :)